spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -980,6 +980,12 @@
       "description": "Whether to enable LDAP repositories.",
       "defaultValue": true
     },
+    {
+      "name": "spring.data.mongodb.additional-hosts",
+      "type": "java.lang.String",
+      "description": "List of additional hosts to connect with different hosts in a replica set.",
```

Comment:
Rather than add to the json, we should be able to add javadoc to the `additionalHosts` field and have the documentation picked up automatically.